test/integration/integration_test.go

```diff
@@ -38,6 +39,8 @@ const (
 	TagIncrementalBuildScripts                = "test/sti-incremental-app-scripts"
 	TagIncrementalBuildScriptsNoSaveArtifacts = "test/sti-incremental-app-scripts-no-save-artifacts"
 	TagCleanLayeredBuildNoTar                 = "test/sti-fake-no-tar"
+	TagCleanBuildOnBuild                      = "test/sti-onbuild-trigger"
+	TagOnBuild                                = "test/sti-file-check-onbuild-trigger"
```

Comment:
these names ("sti-file-check-onbuild-trigger") seem odd.  why not just "test/sti-onbuid-output" and "test/sti-clean-onbuild-output"?

Also in keeping with the others, i'd expect:
TagCleanBuildOnBuild
TagIncrementalBuildOnBuild
to be the variable names.